src/data/checkfiles/default_checkfiles/validate-coord-resc-children.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { Link } from '@reach/router'
+
+export default {
+    name: "Coordinating resources have at least one child",
+    description: `Checks if all coordinating resources (deferred, random, passthru, load_balanced, replication, compound)`,
+    minimum_server_version: "4.2.0",
+    maximum_server_version: "",
+    interval_in_seconds: 3600,
```

Comment:
consider daily?   no strong opinions on this one.

Reply:
Yep, daily seems more reasonable to me here

Reply:
Should we bump the storage one up to daily as well?